qtpy/tests/test_main.py

```diff
@@ -40,6 +46,7 @@ def assert_pyqt5():
         assert QtWebEngineWidgets.QWebEnginePage is PyQt5.QtWebEngineWidgets.QWebEnginePage
     else:
         assert QtWebEngineWidgets.QWebEnginePage is PyQt5.QtWebKitWidgets.QWebPage
+    assert os.environ['QT_API'] == 'PyQt5'
 
 def assert_pyqt6():
```

Comment:
I guess something like `assert os.environ['QT_API'] == 'pyqt6'` is missing for the `assert_pyqt6` function